tests/Mouf/Database/MagicQueryTest.php

```diff
@@ -478,4 +478,16 @@ public function testPrepareStatementCache(): void
         $this->assertEquals('SELECT * FROM users WHERE 0 <> 0', self::simplifySql($magicQuery->buildPreparedStatement('SELECT * FROM users WHERE id IN :users', ['users' => []])));
         $this->assertEquals('SELECT * FROM users WHERE id IN (:users)', self::simplifySql($magicQuery->buildPreparedStatement('SELECT * FROM users WHERE id IN :users', ['users' => [1]])));
     }
+
+    public function testUnionAndOrderBy(): void
+    {
+        $magicQuery = new MagicQuery(null, new ArrayCache());
+
+        $query = '(SELECT id FROM users) UNION (SELECT id FROM users) ORDER BY users.id ASC';
```

Comment:
Should we add another test for

```
(SELECT id FROM users) UNION (SELECT id FROM users ORDER BY users.id ASC)
```

and / or

```
(SELECT id FROM users ORDER BY users.id ASC) UNION (SELECT id FROM users)
```

because maybe we are missing some parentheses here and there.

Reply:
Yes you're right. I added parenthesis around each selects of the UNION
And updated the tests also

Thanks

Reply:
Perfect, thank you !